6_3_iterable.js

```diff
@@ -0,0 +1,48 @@
+class Group {
+    constructor(array) {
+        this.array = array;
+    }
+
+    static from(array) {
+        let group = new Group(array);
+        return group;
+    }
+
+    add(value) {
+        this.array.includes(value) || this.array.push(value);
+        return value;
+    }
+
+    delete(value) {
+        this.array.includes(value) && this.array.splice(this.array.indexOf(value), 1);
+        return value;
+    }
+
+    has(value) {
+        return this.array.some(item => item === value);
+    }
+
+    [Symbol.iterator]() {
+        return new GroupIterator(this);
+    }
+}
+
+class GroupIterator {
+    constructor(cluster) {
+        (this.cluster = cluster), (this.position = 0);
```

Comment:
```
class GroupIterator {
    position = 0
    constructor(cluster) {
        this.cluster = cluster
 ```

U can initialize position like this, futhermore i don't understand  why this 2 commands are in one row and wrapped with braces.